src/main/resources/application.yml

```diff
@@ -17,19 +17,20 @@ spring:
     jpa:
         hibernate:
             ddl-auto: create
-        database-platform: org.hibernate.dialect.MySQL8Dialect
+        database-platform: org.hibernate.dialect.PostgreSQLDialect
         properties:
             hibernate:
                 connection:
                     characterEncoding: utf-8
                     CharSet: utf-8
                     useUnicode: true
+                default_schema: bookcrossing
         open-in-view: false
     datasource:
-        url: jdbc:mysql://localhost:3306/bookcrossing
-        username: root
-        password: root
-        driver-class-name: com.mysql.cj.jdbc.Driver
+        url: jdbc:postgresql://localhost:5432/bookcrossing
```

Comment:
Could you please provide a docker compose file or a docker command in **readme.md** to make postgresql database up and running?